react/Text/Text.js

```diff
@@ -2,6 +2,18 @@ import styles from './Text.less';
 import React, { PropTypes } from 'react';
 import classnames from 'classnames';
 
+import Secondary from '../Secondary/Secondary';
+import Strong from '../Strong/Strong';
+
+const renderContent = ({ children, secondary, strong }) => {
+  const modifiers = [
+    secondary ? Secondary : null,
+    strong ? Strong : null
+  ].filter(x => x);
+
+  return modifiers.reduce((content, mod) => mod({ children: content }), children);
```

Comment:
Could we call it `Modifier` instead of `mod`, and use JSX in the reducer to make it a bit more obvious that we're building up a component tree?

e.g.
```js
return modifiers.reduce((content, Modifier) => <Modifier>{content}</Modifier>, children);
```